microservices-swift/.bluemix/toolchain.yml

```diff
@@ -0,0 +1,62 @@
+version: '2'
```

Comment:
We can probably trash the entire `.bluemix` folder

Reply:
Done here 6116418f68ed172e1a7fa82a96d21ea4c745c4a4